src/constants/theme/colors.ts

```diff
@@ -1,29 +1,25 @@
 export const COLORS = {
-  lunesBlack: '#000e38',
-  lunesWhite: '#f9fafb',
-  lunesGreyMedium: '#5b616a',
-  lunesGreyDark: '#3a4049',
-  white: '#ffffff',
-  lunesBlackUltralight: '#e0e4ed',
-  lunesFunctionalIncorrectDark: '#ff5252',
-  lunesRedLight: '#f57f7a',
-  lunesGreenLight: '#7ae6d8',
-  lunesGreenMedium: '#20c5b4',
-  lunesBlueLight: '#97aef1',
-  lunesOrangeLight: '#ffbb4a',
-  lunesArtikelDas: '#72f399',
-  lunesArtikelDiePlural: '#eee12d',
-  lunesArtikelDie: '#faa7a7',
-  lunesArtikelDer: '#8cc8f3',
-  lunesOverlay: 'rgba(0, 14, 56, 0.8)',
-  lunesBlackLight: '#777e91',
-  blue: '#0000FF',
-  lunesFunctionalCorrectDark: '#0adb75',
-  lunesFunctionalAlmostCorrectDark: '#ffbb4a',
-  lunesRedDark: '#F1635F',
-  lunesRed: '#ffa3a3',
+  primary: '#000e38',
+  backgroundAccent: '#ffffff',
+  background: '#f9fafb',
+  text: '#3a4049',
+  textSecondary: '#5b616a',
+  placeholder: '#777e91',
+  disabled: '#e0e4ed',
+  exerciseProgressIndicator: '#20c5b4',
+  containedButtonSelected: '#3d4662',
+  buttonSelectedSecondary: '#f57f7a',
+  overlay: 'rgba(0, 14, 56, 0.8)',
+  functionalCorrect: '#0adb75',
```

Comment:
Still this functional-prefix. Any reasons for keeping it?

Reply:
nope. will be removed